packages/editor/src/worker/worker.ts

```
@@ -1,138 +1,252 @@
+      switch (data.tag) {
+        case "UpdateReq":
+          log.info("Received UpdateReq in state Optimizing");
+          respondUpdate(stateToLayoutState(optState), stats);
+          break;
+
+        case "ComputeShapesReq":
+          log.info("Received ComputeShapesReq in state Optimizing");
+          const newShapes: LayoutState = stateToLayoutState({
+            ...unoptState,
+            varyingValues: history[data.index],
+          });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)